Tests/test_settings_loader.py

```diff
@@ -19,16 +19,11 @@
 class TestSettingLoader(unittest.TestCase):
 
     def setUp(self):
-        # get current script directory path. We are in /an/unknown/path/kalliope/core/tests
-        cur_script_directory = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
-        # get parent dir. Now we are in /an/unknown/path/kalliope
-        root_dir = os.path.normpath(cur_script_directory + os.sep + os.pardir)
-
-        if "/Tests" in os.getcwd():
-            self.settings_file_to_test = root_dir + os.sep + "settings/settings_test.yml"
+        current_path = os.getcwd()
+        if "/Tests" in current_path:
+            self.settings_file_to_test = current_path + os.sep + "settings/settings_test.yml"
         else:
-            self.settings_file_to_test = root_dir + os.sep + "Tests/settings/settings_test.yml"
-
+            self.settings_file_to_test = current_path + os.sep + "Tests/settings/settings_test.yml"
```

Comment:
This is duplicated on every tests... maybe we could extract it to a static utils tests class ?

Reply:
test utils created and all calls updated.